Changelog for 2.5.3 (#11810)
docker/docker.github.io: docker-for-mac/edge-release-notes.md

For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
[What to know before you install](install.md#what-to-know-before-you-install).

## Docker Desktop Community 2.5.3
2020-11-30

> [Download](https://desktop.docker.com/mac/edge/50253/Docker.dmg)

### Upgrades

- [Compose CLI v1.0.3](https://github.com/docker/compose-cli/releases/tag/v1.0.3)

### Bug fixes and minor changes

> - Fix a possible filesharing glitch where a file may appear to have the wrong size in a container when modified on the host. Partial fix for [docker/for-mac#4999](https://github.com/docker/for-mac/issues/4999).

Reviewer: ```suggestion
- Fixed a possible file sharing error where a file may appear to have the wrong size in a container when it is modified on the host. This is a partial fix for [docker/for-mac#4999](https://github.com/docker/for-mac/issues/4999).
```

> - Remove unnecessary log messages which slow down filesystem event injection.

Reviewer: ```suggestion
- Removed unnecessary log messages which slow down filesystem event injection.
```
